Xamarin.Essentials/Calendar/Calendar.uwp.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Appointments;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Calendar
+    {
+        static async Task<IEnumerable<DeviceCalendar>> PlatformGetCalendarsAsync()
+        {
+            await Permissions.RequireAsync(PermissionType.CalendarRead);
+
+            var instance = await CalendarRequest.GetInstanceAsync();
+            var uwpCalendarList = await instance.FindAppointmentCalendarsAsync(FindAppointmentCalendarsOptions.IncludeHidden);
+            var calendars = new List<DeviceCalendar>();
+            foreach (var c in uwpCalendarList)
```

Comment:
linq

Reply:
Addressed in:
https://github.com/builttoroam/Essentials/pull/43